### ISSUE TITLE
Lab# 7 - Misleading project name

### ISSUE THREAD

**Issue by kirhie** [mglantz/ansible-roadshow#154]
**Describe the bug**

**To Reproduce**
1. Go to 'Lab 7'
2. Scroll down to section where we create new project in Ansible Tower "workshop-playbooks"
4. See error - The screen print below shows project name as "workshop-inventory" which is a bit misleading and confuses with the inventory name.

**Expected behavior**
The project name should be same in description and screen print.
![proj-name](https://user-images.githubusercontent.com/20984600/45095990-beeb6300-b11f-11e8-8cdf-0a6c58db9870.JPG)

**Comment by mglantz** [comment]
I agree, graphics and text should match. Can you create a pull request for this?
